content/posts/November-2020.md

```diff
@@ -23,16 +23,35 @@ draft: false
 
 ## Week 47 (November 16th - November 20th)
 
-- [Matej] fixed [an
+- ~~[Matej] fixed [an
   issue](https://github.com/packit/packit-service/issues/774) with Packit CLI,
   enabling recursive search for spec-files
-  ([packit#1005](https://github.com/packit/packit/pull/1005)).
+  ([packit#1005](https://github.com/packit/packit/pull/1005)).~~ It turned
+  out, this change made it very easy to [deplete GitHub API quota], so it was
+  reverted.
 - [Laura] made Copr builds to be part of the default jobs. This will cause
   build jobs to explicitly run, and display their own result flags in PRs,
   next to the test results. We hope this to be a more friendly behaviour for
   our new users ([packit#1024](https://github.com/packit/packit/pull/1024)).
 
+## Week 48 (November 23rd - November 27th)
+
+- [Laura] [fixed a bug], where Packit Service failed to trigger Copr builds
+  before triggering the tests in case triggering Copr builds was configured
+  for a different event.
+- Thanks to [Matej], Packit Service will now [comment on commits] in case Copr
+  builds fail after a merge, in order to indicate the reason for the failure.
```

Comment:
@mfocko did I put it the right way? :slightly_smiling_face:

Reply:
Yes :+1: